src/App.tsx

```diff
@@ -29,128 +53,29 @@ export const App: React.FC = () => {
-            </div>
-          </div>
-
-          {/* This todo is being edited */}
-          <div className="todo">
-            <label className="todo__status-label">
-              <input
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            {/* This form is shown instead of the title and remove button */}
-            <form>
-              <input
-                type="text"
-                className="todo__title-field"
-                placeholder="Empty todo will be deleted"
-                value="Todo is being edited now"
-              />
-            </form>
-
-            <div className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-
-          {/* This todo is in loadind state */}
-          <div className="todo">
-            <label className="todo__status-label">
-              <input type="checkbox" className="todo__status" />
-            </label>
-
-            <span className="todo__title">Todo is being saved now</span>
-            <button type="button" className="todo__remove">×</button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+          {
+            listTodo.map((el: Todo) => <Item todo={el} key={el.id} />)
+          }
         </section>
 
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {listTodo.length > 0
+          ? (
+            <footer className="todoapp__footer">
+              <span className="todo-count">
+                {`${listTodo.length} items left`}
+              </span>
+
+              <Filter setFilter={getListTodo} />
+
+              {/* don't show this button if there are no completed todos */}
+              <button type="button" className="todoapp__clear-completed">
+                Clear completed
+              </button>
+            </footer>
+          ) : ''}
```

Comment:
Can be simplify
And remove comments
```suggestion
        {listTodo.length > 0
          && (
            <footer className="todoapp__footer">
              <span className="todo-count">
                {`${listTodo.length} items left`}
              </span>

              <Filter setFilter={getListTodo} />

              <button type="button" className="todoapp__clear-completed">
                Clear completed
              </button>
            </footer>
          )}
```